### PR TITLE
Update webpack.config.js

### PR DESCRIPTION
**Pull request by seyyidenurunal**

### CODE REVIEW

**Comment by unalsemih** [comment]
test comment

File: webpack.config.js [hunk 1]
```diff
@@ -7,4 +7,5 @@ module.exports = {
     filename: 'main.js'
   },
   watch: false
-};
\ No newline at end of file
+};
```
Comment (unalsemih) [inline webpack.config.js:10]: ..

**Review by unalsemih — COMMENTED** [review]
.

**Review by unalsemih — COMMENTED** [review]
..

File: webpack.config.js (hunk 1 above)
Comment (seyyidenurunal) [reply webpack.config.js:10]: .

Comment (seyyidenurunal) [reply webpack.config.js:10]: @unalsemih s

**Comment by seyyidenurunal** [comment]
@unalsemih s2

**Pull request closed without merging**